js/apps/admin-ui/maven-resources/theme/keycloak.v2/admin/messages/messages_en.properties

```diff
@@ -3579,6 +3579,18 @@ oid4vciNonceLifetimeHelp=The lifetime of the OID4VCI nonce in seconds.
 preAuthorizedCodeLifespan=Pre-Authorized Code Lifespan
 preAuthorizedCodeLifespanHelp=The lifespan of the pre-authorized code in seconds.
 oid4vciFormValidationError=Please ensure the OID4VCI attribute fields are filled with values 30 seconds or greater.
+# OID4VCI Credential Configuration
+credentialConfigurationId=Credential Configuration ID
+credentialConfigurationIdHelp=The unique identifier for this credential configuration. This ID is used in the credential issuer metadata and credential requests.
+credentialIdentifier=Credential Identifier
+credentialIdentifierHelp=A specific identifier for this credential type. This can be used to distinguish between different variants of the same credential type.
+issuerDid=Issuer DID
+issuerDidHelp=The Decentralized Identifier (DID) of the credential issuer. This identifies who is issuing the verifiable credentials.
+credentialLifetime=Credential Lifetime (seconds)
+credentialLifetimeHelp=The lifetime of the credential in seconds. After this time, the credential will expire and become invalid.
+supportedFormats=Supported Formats
+supportedFormatsHelp=The format of the verifiable credential. Currently supported formats: SD-JWT VC (dc+sd-jwt), JWT VC (jwt_vc_json).
```

Comment:
This looks unrelated to the workflows change - these entries are already in present in current main

Reply:
Ugh - looks like I was not paying attention when I reviewed.. sorry!